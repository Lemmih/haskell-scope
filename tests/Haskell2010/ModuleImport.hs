-- Must check:
--  no import spec
--  IVar
--  IAbs
--  IThingAll
--  IThingWith
module ModuleImport where

import Stdlib (Int)
import Stdlib (Maybe(..))

type MyInt = Int
