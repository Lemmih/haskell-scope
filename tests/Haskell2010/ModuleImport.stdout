Scope errors:

Definitions:
  Definition 1, from ModuleImport.decl0:
    type [4mMyInt[0m = Int

Use sites:
  Definition used: Stdlib.decl1
    import Stdlib ([4mInt[0m, fromJust)
  Definition used: Stdlib.decl6
    import Stdlib (Int, [4mfromJust[0m)
  Definition used: Stdlib.decl2
    import Stdlib ([4mMaybe[0m(..), Bool(True))
  Definition used: Stdlib.decl3
    import Stdlib (Maybe(..), [4mBool[0m(True))
  Definition used: Stdlib.decl3
    import Stdlib (Maybe(..), Bool([4mTrue[0m))
  Definition used: Stdlib.decl1
    type MyInt = [4mInt[0m

Exports:
  MyInt, Type from ModuleImport
